### ISSUE TITLE
ol-solr0 space filling up

### ISSUE THREAD

**Issue by mekarpeles** [internetarchive/openlibrary#3778]
### Summary

- **What is wrong?**

Noticed NAGIOS complaining about storage being low on ol-solr0

- **What caused it?**

Docker logs on ol-solr0 have no limit. Also tomcat7 logs inside the docker container.

- **What fixed it?**

Clearing the docker images logs:

```sh
truncate -s 0 $(docker inspect --format='{{.LogPath}}' d12b518475e1)
```

Note `d12b518475e1` is the id of the solr docker container and will change; run `docker container ls` to find the current one's id

As well as the tomcat logs inside the docker container at `/var/log/tomcat7`.

- **Followup actions:**
 

Limit the size of the logs, and re-deploy ol-solr0. #3720 

### Steps to close

1. [x] Assignment: Is someone assinged to this issue? (notetaker, responder)
2. [x] Labels: Is there an `Affects:` label applied?
3. [x] Diagnosis: Add a description and scope of the issue
4. [x] Updates: As events unfold, is noteable **provenance** documented in issue comments? (i.e. useful debug commands / steps / learnings / reference links)
5. [x] "What caused it?" - please answer in summary
6. [x] "What fixed it?" - please answer in summary
7. [x] "Followup actions:" actions added to summary

**Comment by cdrini** [comment]
Here are some more useful commands:

Check sizes (the docker ones are the ones that fill up):
```sh
$ sudo df -h
Filesystem      Size  Used Avail Use% Mounted on
...
overlay         237G  213G   12G  95% /var/lib/docker/overlay/3b16ff84.../merged
...
```

Find the container id:
```sh
$ sudo docker container ls
CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS                    NAMES
d12b518475e1        olsolr              "/bin/sh -c /start.sh"   7 weeks ago         Up 7 weeks          0.0.0.0:8983->8983/tcp   mystifying_noyce
```

Truncate the logs (Or `sudo bash` to avoid `sudo`ing everywhere)

```sh
$ time sudo truncate -s 0 $(sudo docker inspect --format='{{.LogPath}}' d12b518475e1)

real    0m3.877s
user    0m0.115s
sys     0m0.053s
```

Check it worked:

```sh
drini@ol-solr0:~$ sudo df -h
Filesystem      Size  Used Avail Use% Mounted on
...
overlay         237G  203G   23G  91% /var/lib/docker/overlay/3b16ff84.../merged
...
```

**Comment by cdrini** [comment]
It looks like it was also the tomcat logs, which were stored inside the docker image separately (need to find a way to disable those). ol-solr2 is only at 40G of 237G; ol-solr0 was at 230G of 237G!

I `du -sh *` deeper into `/var/lib/docker/overlay/3b16ff84.../merged` until I found the culprit: `/var/log/tomcat7` - `145G`

I cleared out any old files.

This brought it down to 90G of 237G.

There's still something funny going on here, because when I directly `df -sh` the line from `df -h`, it shows up as _27G_, not 90G.
